composer.json

```diff
@@ -20,7 +20,9 @@
     "require": {
         "php": "^7.2",
         "cocur/slugify": "^1.4 || ^2.0 || ^3.0 || ^4.0",
-        "sonata-project/doctrine-extensions": "^1.1",
+        "sonata-project/exporter": "^1.11 || ^2.0",
+        "sonata-project/form-extensions": "0.x-dev",
```

Comment:
RTM, but we should require stable releases

Reply:
https://github.com/sonata-project/SonataCoreBundle/pull/736#issuecomment-629285760

I wrote about it.

Reply:
Done.